### PR TITLE
Finalize Path Generation

### PR DESCRIPTION
**Pull request by marv7000**
# What does this PR change?

Tick the applicable box:
- [X] Add new feature
- [ ] Security changes
- [ ] Tests
- [ ] Documentation changed
<br/>

- [ ] General Maintenance

## Links

Fixes: #14 

- [x] DONE

## Documentation

- No documentation needed
<br/>

- [x] DONE

### CODE REVIEW

**Review by ByteOtter — APPROVED** [review]
LGTM. Next We need to expand the documentation for the functions. I will do this later.

File: src/templates/util.rs.template
Comment (ByteOtter) [inline src/templates/util.rs.template:4]: The URL needs to be injected via the CLI or by Nazara itself. Statically changing it here does not work.

**Review by ByteOtter — CHANGES_REQUESTED** [review]

File: src/templates/util.rs.template
Comment (ByteOtter) [reply src/templates/util.rs.template]: I fixed it

**Review by ByteOtter — APPROVED** [review]

**Review by ByteOtter — APPROVED** [review]

**Review by SchoolGuy — APPROVED** [review]

**Comment by ByteOtter** [comment]
I am going to merge this without waiting for another review.
Thanks to you, Thanix can now successfully generate code from a api spec yaml and build a crate from it. Therefore, my acceptance criteria are fulfilled.
Should we find any flaws in this implementation later or should the resulting crate not work with Nazara (or NetBox for taht matter), we can always go back and fix it in another PR.

**Pull request merged**
